plan/new_physical_plan_builder.go

```diff
@@ -743,34 +737,28 @@ func (p *DataSource) convertToIndexScan(prop *requiredProp, idx *model.IndexInfo
 		}
 	}
 	is.profile = p.getStatsProfileByFilter(p.pushedDownConds)
+	setHandle := is.initSchema(p.id, idx)
```

Comment:
Can we move this function down to 762?
So we can pass a `isDobuleRead` argument `cop.tablePlan != nil` to `initSchema` function, remove the return value.
`init` function should not return anything except for error.

Reply:
And the init schema work better be done in one place, now it is done in two places.